great_expectations/expectations/core/expect_column_values_to_be_dateutil_parseable.py

```diff
@@ -74,7 +74,7 @@ class ExpectColumnValuesToBeDateutilParseable(ColumnMapExpectation):
         "requirements": [],
     }
 
-    map_metric = "column_values.dateutil_parsable"
+    map_metric = "column_values.dateutil_parseable"
```

Comment:
Ah great catch! Not in the scope of this current PR but should we have a check somewhere that validates strings like this? I would think that typos are quite common. Perhaps raising an error if the map_metric isn't within a list/set of accepted metrics?

I don't know enough about this part of the codebase though so perhaps I'm lacking the requisite knowledge.

Reply:
Thanks, @cdkini! It would be great to have tests like that, though given the way we pass in metrics as strings, I'm not sure of the best way to do this. Maybe a step where when we register an expectation, we also check that the map_metric for that expectation is registered? 

FWIW, the json test added here would have caught this, we just don't yet have full coverage with all those tests. That might be a worthwhile endeavor at some point.

Reply:
Do we have an inventory of map expectations that aren't test-covered here? That seems like a good story to get into a sprint pretty soon; should be pretty low impact and nice to make sure we're clean going into v3.

Reply:
Hmm I'd have to do some research and see all the places it is actually used but that does sound quite reasonable! And I agree with @fjork3 that would be quite helpful!

Maybe some more items for the backlog 👀